src/test/java/io/jenkins/plugins/conventionalcommits/ConventionalCommitsTest.java

```diff
@@ -79,4 +82,60 @@ public void willBumpMajorVersion_MultipleCommits() {
         assertThat(out, is(notNullValue()));
         assertThat(out.toString(), is("1.0.0"));
     }
+
+    @Test
+    public void willBumpMajorVersion_ExclamationCommit() {
+        ConventionalCommits cc = new ConventionalCommits();
+
+        Version out = cc.nextVersion(Version.valueOf("0.0.1"), Collections.singletonList(
+                "feat!: new major version"));
+        assertThat(out, is(notNullValue()));
+        assertThat(out.toString(), is("1.0.0"));
+    }
+
+    @Test
+    public void willBumpMajorVersion_FooterMultipleLineCommit() {
+        ConventionalCommits cc = new ConventionalCommits();
+
+        Version out = cc.nextVersion(Version.valueOf("0.0.1"), Collections.singletonList(
+                "feat: new major version \nBREAKING CHANGE: new breaking change"
+        ));
+        assertThat(out, is(notNullValue()));
+        assertThat(out.toString(), is("1.0.0"));
+    }
+
+    @Test
+    public void willNotBumpMajorVersion_ExclamationMultipleLineCommitNotFooter() {
+        ConventionalCommits cc = new ConventionalCommits();
+
+        Version out = cc.nextVersion(Version.valueOf("0.0.1"), Collections.singletonList(
+                "chore: new major version \nBREAKING CHANGE: new breaking change \nstupid footer"
+        ));
+        assertThat(out, is(notNullValue()));
+        assertThat(out.toString(), is("0.0.2"));
+    }
```

Comment:
From what I've [read](https://www.conventionalcommits.org/en/v1.0.0/#summary), a BREAKING CHANGE can be a part of commits of any type.

So can we say for sure whether the footer is "stupid" or the user actually wanted to bump the version with an extra "chore" commit?

Reply:
`BREAKING CHANGE:` must be the last line of the commit message to be considered as a breaking change. Otherwise it's just a random commit description.

By ending the commit message with `stupid footer` @jonyto123 want to test that it was just a patch update

Reply:
I see, the only reason I asked that question was I saw breaking change need not be the last line 😅 
Then again, I may not have read thoroughly, I am talking about [this discussion](https://github.com/conventional-commits/conventionalcommits.org/issues/179) on the conventional-commits GitHub.